go/ct/common/hash256.go

```diff
@@ -0,0 +1,116 @@
+//
+// Copyright (c) 2024 Fantom Foundation
+//
+// Use of this software is governed by the Business Source License included
+// in the LICENSE file and at fantom.foundation/bsl11.
+//
+// Change Date: 2028-4-16
+//
+// On the date above, in accordance with the Business Source License, use
+// of this software will be governed by the GNU Lesser General Public Licence v3
+//
+
+package common
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/Fantom-foundation/Tosca/go/vm"
+	"pgregory.net/rand"
+)
+
+// Hash256 is an immutable array of Hash256 that can be trivially cloned.
+type Hash256 struct {
```

Comment:
nitpick: The type contains more than one hash, name should be plural. 
Comment should say array of 256 Hashes as well.
Why not BlockHashHistory or LastBlockHashes?

Reply:
I think those names convey meaning of a particular use, while this is just an immutable array. I would rather call the type `Hashes256` and fix the comment, rather than talk about blocks here.